disconnected/install-disconnected.adoc

```diff
@@ -0,0 +1,26 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="install-disconnected"]
```

Comment:
```suggestion
[id="install-disconnected_{context}"]
```

Reply:
Pretty sure this is only for [modules](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata). This one is an assembly.